src/core/__tests__/template.test.tsx

```diff
@@ -87,6 +87,29 @@ describe("function template", () => {
         expect(component.find(".template").html()).toBe('<div class="template">Template with data</div>');
     });
 
+    it("renders template wrapper", () => {
+        const itemRender: any = jest.fn((text: string) => <div>Template {text}</div>);
+        const component = mount(
+            <ComponentWithTemplates itemRender={itemRender} />
+        );
+        const container = document.createElement("div");
+        renderItemTemplate("with data", container);
+        component.update();
```

Comment:
Why `.update()` is essential?

Reply:
It is necessary to update the component state to render template completely.